packages/babel-plugin-transform-parameters/src/rest.js

```diff
@@ -305,7 +305,8 @@ export default function convertFunctionRest(path) {
     return true;
   }
 
-  state.references = state.references.concat(
+  Array.prototype.push.apply(
+    state.references,
     state.candidates.map(({ path }) => path),
   );
```

Comment:
This seems changed behavior.

Reply:
How could it be? It takes the result of `.map`🤔

Reply:
Ah , sorry, I thought it is `array.push()`